src/react/databrowser/buckets/details/Overview.tsx

```diff
@@ -103,17 +108,37 @@ function Overview({ bucket, ingestionStates }: Props) {
   const { account } = useCurrentAccount();
   const [isErrorModalOpen, setIsErrorModalOpen] = useState(false);
   const [bucketTaggingToast, setBucketTaggingToast] = useState(true);
+  const [isCapacityModalOpen, setIsCapacityModalOpen] = useState(false);
   const { tags } = useBucketTagging({ bucketName: bucket.name });
   const VEEAM_FEATURE_FLAG_ENABLED = features.includes(VEEAM_FEATURE);
-  const isVeeamBucket =
+  const veeamTagApplication =
     tags.status === 'success' &&
-    (tags.value?.[BUCKET_TAG_VEEAM_APPLICATION] === VEEAM_BACKUP_REPLICATION ||
-      tags.value?.[BUCKET_TAG_VEEAM_APPLICATION] === VEEAM_OFFICE_365) &&
+    decodeEntities(tags.value?.[BUCKET_TAG_VEEAM_APPLICATION]);
+  const isVeeamBucket =
+    (veeamTagApplication === VeeamApplicationType.VEEAM_BACKUP_REPLICATION ||
+      veeamTagApplication === VeeamApplicationType.VEEAM_OFFICE_365) &&
     VEEAM_FEATURE_FLAG_ENABLED;
 
   const isSOSAPIEnabled =
     isVeeamBucket &&
-    tags.value?.[BUCKET_TAG_VEEAM_APPLICATION] === VEEAM_BACKUP_REPLICATION;
+    veeamTagApplication === VeeamApplicationType.VEEAM_BACKUP_REPLICATION;
+
+  const s3Client = useS3Client();
+  const { data: veeamObject } = useQuery(
+    getVeeamObject({
+      bucketName: bucket.name,
+      s3Client,
+    }),
+  );
+
+  const xml = veeamObject?.Body?.toString();
+  const regex = /<Capacity>([\s\S]*?)<\/Capacity>/;
+  const matches = xml?.match(regex);
+  const capacity = matches ? matches[1] : '0';
+  const prettyBytesClusterCapacity = prettyBytes(parseInt(capacity, 10), {
+    locale: 'en',
+    binary: true,
+  });
```

Comment:
I would vote for extracting all the related code for the capacity to it's own component, I feel that it is going to be easier

Reply:
Fixed in 49e7d8dae934e9182e815b5ad0f47c51712c374b